cmd/thanos/sidecar.go

```diff
@@ -284,14 +292,23 @@ func (s *extLabelSet) Update(ctx context.Context) error {
 	return nil
 }
 
-func (s *extLabelSet) Get() labels.Labels {
+func (s *metadata) UpdateTimestamps(mint int64, maxt int64) error {
+	s.mtx.Lock()
+	s.mint = mint
+	s.maxt = maxt
+	s.mtx.Unlock()
+
+	return nil
+}
+
+func (s *metadata) GetLabels() labels.Labels {
```

Comment:
Is it not normally convention to have `Labels()` and no `Get` methods in Go?

Reply:
Let's do it